dist/docs_data.py

```diff
@@ -0,0 +1,25 @@
+# Create entries used by our doxygen filter to expand the arch_page
+# macros in the documentation.
+
+class ArchDocPage:
+    def __init__(self, doxygen_name, data_structures, files):
+        self.doxygen_name = doxygen_name
+        self.data_structures = data_structures
+        self.files = files
+
+##########################################
+# List of all architecture subsections
+##########################################
+arch_doc_pages = [
+    ArchDocPage('arch-schema',
+        ['WT_TABLE', 'WT_COLGROUP', 'WT_INDEX'],
```

Comment:
Is this meant to be complete or just a quick example for the diagram? If complete, it is missing LSM.

Reply:
This page was done pretty quickly, but we might as well fix this and add LSM.  I'm expecting that different people will 'own' each section, and fill out the pieces.